beacon_node/beacon_chain/src/data_availability_checker.rs

```diff
@@ -412,14 +415,11 @@ impl<T: BeaconChainTypes> DataAvailabilityChecker<T> {
 
         let all_data_columns = blocks
             .iter()
-            .filter(|block| self.data_columns_required_for_block(block.as_block()))
```

Comment:
I don't like that we have different filtering for blobs and columns verification here. 
its not expected from the name of the function or the docs, its just a temporary hack to get a test passing.
If we want to deal with this later, i'd rather we merge this without the hack and deal with the failing test before merging peerdas-devnet-7 to unstable. 
cc @jimmygchen

Reply:
I think now tests should pass still with that line. I'll add it back to this PR.

I'll make an issue to remove this line in unstable as I think it's quite problematic

Reply:
Awesome, thanks!